helpers/configuration.py

```diff
@@ -42,59 +44,38 @@ def load_config(self, extra_zap_params):
             self.extra_zap_params = extra_zap_params
             log(f"Extra params passed by ZAP: {self.extra_zap_params}")
 
-            self.auth_login_url = self._get_zap_param('auth.loginurl') or EMPTY_STRING
-            self.auth_username = self._get_zap_param('auth.username') or EMPTY_STRING
-            self.auth_password = self._get_zap_param('auth.password') or EMPTY_STRING
-            self.auth_otp_secret = self._get_zap_param('auth.otpsecret') or EMPTY_STRING
-            self.auth_submit_action = self._get_zap_param('auth.submit_action') or 'click'
-            self.auth_form_type = self._get_zap_param('auth.form_type') or 'simple'
-            self.auth_token_endpoint = self._get_zap_param('auth.token_endpoint') or EMPTY_STRING
-            self.auth_bearer_token = self._get_zap_param('auth.bearer_token') or EMPTY_STRING
-            self.auth_username_field_name = self._get_zap_param('auth.username_field') or 'username'
-            self.auth_password_field_name = self._get_zap_param('auth.password_field') or 'password'
-            self.auth_display = self._get_zap_param('auth.display') or EMPTY_STRING
-            self.auth_otp_field_name = self._get_zap_param('auth.otp_field') or 'otp'
-            self.auth_submit_field_name = self._get_zap_param('auth.submit_field') or 'login'
-            self.auth_submit_second_field_name = self._get_zap_param('auth.second_submit_field') or 'login'
-            self.auth_delay_time = self._get_zap_param_float('auth.delay_time') or 0
-            self.auth_check_delay = self._get_zap_param_float('auth.check_delay') or 5
-            self.auth_check_element = self._get_zap_param('auth.check_element') or EMPTY_STRING
-            self.auth_exclude_urls = self._get_zap_param_list('auth.exclude') or list()
-            self.auth_include_urls = self._get_zap_param_list('auth.include') or list()
-            self.xss_collector = self._get_zap_param('xss.collector') or EMPTY_STRING
-            self.cookies = self._get_zap_param('request.custom_cookies') or EMPTY_STRING
-            self.header = self._get_zap_param('request.custom_header') or EMPTY_STRING
-            self.oauth_token_url = self._get_zap_param('oauth.token_url') or EMPTY_STRING   
-            self.oauth_parameters = self._get_zap_param_list('oauth.parameters') or EMPTY_STRING
+            self.auth_login_url = os.environ.get('AUTH_LOGIN_URL') or EMPTY_STRING
```

Comment:
previously all of these params were passed to the zap tool eg 
`python3 /zap/zap-full-scan.py -t targeturl.com -d -j -z \" auth.loginurl='https://targeturl.com/login.jsp' auth.username='********' auth.password='********' auth.form_type='simple' auth.delay_time='5' \" --hook /zap/hooks/soos_dast_hook.py -J report.jso` 
all of these params were included on the -z argument that is intender for "other zap params" like plugin updates. Appending values in there could cause issues where they wouldn't be parsed properly/truncated and it caused the scan to throw for that reason.